Adds `UVFunc` for panning/scaling on UV's to VisualShader's.

Chaosus opened the pull request:
+ Adds new node called `UVFunc` which contains two functions: Panning and Scaling:

![vs_texture_func](https://user-images.githubusercontent.com/3036176/120895616-793c8b80-c626-11eb-97c3-672ad3022a31.gif)

+ Also adds `clear_default_input_values` and `remove_input_port_default_value` to `VisualShaderNode` to clear/remove default input values - this new node will use it ^^ (to clear default value of 2nd port on switching).

This is my reaction to a user request on https://github.com/godotengine/godot-proposals/issues/2791 / https://github.com/godotengine/godot-proposals/issues/2791#issuecomment-850846697 I really don't know how useful it would be but I decided to do it.

Chaosus committed:
Adds `UVFunc` for panning/scaling on UV's to VisualShader's.

clayjohn commented:
For panning, I think it makes sense for the ``time`` input to be ``scale`` and let users specify the input (defaulting to 1). I think the panning over time is a unique use case. Also, users can easily feed time in as ``scale`` and it will work properly

Chaosus commented:
@clayjohn Thanks, fixed: changed the time to scale + changed it to vector type + changed it to 2nd port (instead of 3rd), removed TIME as the default value for it

![image](https://user-images.githubusercontent.com/3036176/120964143-7ac39c00-c76b-11eb-86e7-3e707ce196eb.png)

akien-mga commented:
Thanks!
